### PR TITLE
Promote mount propagation to GA

### PR DESCRIPTION
**Pull request by bertinatto**

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -1236,8 +1236,6 @@ More details can be found [here](https://github.com/kubernetes/community/blob/ma
 
 ## Mount propagation
 
-{{< feature-state for_k8s_version="v1.10" state="beta" >}}
-
 Mount propagation allows for sharing volumes mounted by a Container to
 other Containers in the same Pod, or even to other Pods on the same node.
```
Comment (msau42) [inline content/en/docs/concepts/storage/volumes.md]: Also the line below that mentions the feature gate should be removed

**Comment by zparnold** [comment]
/hold 
I'd like to place a hold so this isn't inadvertently closed before the 1.12 release date

**Comment by bertinatto** [comment]
Superseded by #10090.

**Pull request closed without merging**